.github/workflows/build-tokens.yaml

```diff
@@ -0,0 +1,71 @@
+name: Build Tokens
+on:
+  pull_request:
+    types: [opened, synchronize, edited, reopened]
```

Comment:
Is this necessary to run the workflow when the PR id edited as well?

Reply:
I'm not even sure if it is necessary at all.
What do you think?

Reply:
The changeset magic force pushes to the changeset PR instead of doing a regular push; therefore, the synchronize trigger does not activate. Fortunately, changeset magic also updates the PR description, which allowed us to use the edited trigger as a workaround to ensure the documentation preview remains up-to-date after a force push on this PR.

If the build token summary needs to be updated with any change on the changeset PR, then you can leave the triggers as-is. Otherwise, you can fully remove the types.